tests/unit/amo/components/TestReportAbuseButton.js

```diff
@@ -81,101 +85,54 @@ describe(__filename, () => {
   });
 
   it('shows more content when the "report" button is clicked', () => {
+    const addon = { ...fakeAddon, slug: 'my-addon-show-UI' };
     const fakeEvent = createFakeEvent();
+    const { store } = dispatchClientMetadata();
+    const dispatchSpy = sinon.spy(store, 'dispatch');
     // We need to use mount here because we're interacting with refs. (In
     // this case, the textarea.)
-    const root = renderMount();
```

Comment:
I agree the comment is not accurate any more. I tried `shallow`, but it seems that I need to call `renderShallow` again to make the `expect` line works. I am not sure why it does not rerender. If calling `renderShallow` twice is ok, I will change it to `shallow`.
```
let root = renderShallow({ addon, store });
root.find(Button).simulate('click', fakeEvent)
root = renderShallow({ addon, store });                                     
expect(root.find('.ReportAbuseButton--is-expanded')).toHaveLength(1);    
```

Reply:
what if you call `root.update()` instead?

Reply:
Neither of `root.update()` / `root.instance().forceUpdate()` works. It looks like `root` (`<ReportAbuseButtonBase/>`) cannot get the updated props value from `<ReportAbuseButton>`. It might be related to https://github.com/airbnb/enzyme/issues/1229#issuecomment-413718414

I found some other tests using `render` twice. https://github.com/mozilla/addons-frontend/blob/25e8a36dda1dadc40a0363893242d1ced110cedc/tests/unit/amo/components/TestCollectionDetailsCard.js#L262
https://github.com/mozilla/addons-frontend/blob/25e8a36dda1dadc40a0363893242d1ced110cedc/tests/unit/amo/components/TestAddonReviewCard.js#L1003

Reply:
I see. let's keep it as is then.

Could you update the other parts?